packages/volto/locales/volto.pot

```diff
@@ -3987,9 +3987,9 @@ msgstr ""
 msgid "Type"
 msgstr ""
 
-#. Default: "Type a Video (YouTube, Vimeo or mp4) URL"
+#. Default: "Type a valid URL"
 #: components/manage/Blocks/Video/Edit
-msgid "Type a Video (YouTube, Vimeo or mp4) URL"
+msgid "Type a valid URL"
```

Comment:
Changing the `msgid` may be a breaking change. It's merely an identifier and is not exposed to the end user. You may change the `Default` and `msgstr` without making a breaking change. See https://github.com/plone/volto/issues/6159

Reply:
ok